keyboards/handwired/t111/t111.h

```diff
@@ -0,0 +1,57 @@
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define XXX KC_NO
+
+/* 
+,-------------------------------------------------------------------------------------------------------------------------------------.
+|break|     | PF1 | PF2 | PF3 | PF4 | PF5 | PF6 | PF7 | PF8 | PF9 | PF10| PF11| PF12| PF13| PF14| PF15| PF16|                         |
+|-------------------------------------------------------------------------------------------------------------------------------------|
+| esc |  1  |  2  |  3  |  4  |  5  |  6  |  7  |  8  |  9  |  0  |  -  |  =  |  ~  |backspc|  | ins | cls |  |  *  |  /  |  +  |  -  |
+|-------------------------------------------------------------------------------------------------------------------------------------|
+|  tab  |  Q  |  W  |  E  |  R  |  T  |  Y  |  U  |  I  |  O  |  P  |  [  |  ]  |   enter      |  el | dup |  |  7  |  8  |  9  |  =  |
+|--------------------------------------------------------------------------------              |--------------------------------------|
+|  ctrl  |  A  |  S  |  D  |  F  |  G  |  H  |  J  |  K  |  L  |  ;: |  '" |  |\ |             | del | home|  |  4  |  5  |  6  |  ,  |
+|-------------------------------------------------------------------------------------------------------------------------------------|
+|   shift   |  Z  |  X  |  C  |  V  |  B  |  N  |  M  |  ,  |  .  |  /? |blank|     shift      |  up | down|  |  1  |  2  |  3  |enter|
+|-------------------------------------------------------------------------------------------------------------------------------|     |
+|         | caps| graph |                    Space                      | alt |                | left|right|  |     0     |  .  |     |
+`-------------------------------------------------------------------------------------------------------------------------------------'
+*/
+
+#define LAYOUT( \
+  K30,      K40, K11, K21, K31, K41, K51, K61, K71, K01, K12, K22, K32, K42, K52, K62, K72, \
+  K20, K13, K23, K33, K43, K53, K63, K73, K03, K14, K24, K34, K44, K54, K58,   K64, K74,   K15, K06, K76, K66, \
+  K10,  K17, K27, K37, K47, K57, K67, K77, K07, K18, K28, K38, K48,     K5A,   K68, K78,   K1B, K00, K70, K60, \
+  K0A,   K19, K29, K39, K49, K59, K69, K79, K09, K1A, K2A, K3A, K4A,           K6A, K7A,   K2B, K3B, K4B, K5B, \
```

Comment:
QMK custom is that ISO Enter's location is toward the end of the home row.

```suggestion
  K10,  K17, K27, K37, K47, K57, K67, K77, K07, K18, K28, K38, K48,            K68, K78,   K1B, K00, K70, K60, \
  K0A,   K19, K29, K39, K49, K59, K69, K79, K09, K1A, K2A, K3A, K4A,    K5A,   K6A, K7A,   K2B, K3B, K4B, K5B, \
```